AddieC/.angular-cli.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "project": {
+    "name": "addie-c"
+  },
```

Comment:
Yes, the .gitignore is added. We may want to go through and update it according to what fits our project best.